README.md

```diff
@@ -43,6 +43,11 @@ config: {
   // When set, response which invalidates token will be resolved after the token has been renewed
   // in effect, token will be loaded in sync with response, otherwise renew will run async to response
   shouldWaitForTokenRenewal: boolean,
+  
+  // Checks if response should be considered unauthorized (by default only 401 responses are 
+  // considered unauthorized). Override this method if you need to trigger token renewal for 
+  // other response statuses.
```

Comment:
I should add extra info about are you overriding or composing. Because you are replacing behavior with default one. It would be great if I could override `isResponseUnauthorized` but also import from library default function so I can create composition.